doc/api/examples/codeRepository2.json

```diff
@@ -0,0 +1,11 @@
+{
+    "codeRepositoryId": "idcode2",
```

Comment:
IDs are now defined as strings in UUID format. You can generate them with
`python -c "import uuid; print(uuid.uuid4())"`